services/web/client/source/resource/form/service-data.json

```diff
@@ -0,0 +1,50 @@
+{
+    "contributor": {
+        "firstName": "Ignacio",
+        "lastName": "Pascual",
+        "email": "pascual@itis.swiss",
+        "affiliation": "Universidad Politécnica de Valencia"
+    },
+    "serviceInfo": {
+        "description": "Universal solver that solves everything",
+        "commandLine": "In linux terminal: python3 universalsolver.py input1 input2",
+        "url": "example.github.com",
+        "codeVariant": "python"
+    },
+    "serviceInterface": {
+        "inputs": [
+            {
+                "name": "input1",
+                "type": "complex",
+                "defaultValue": "null",
+                "label": "input #1",
+                "description": "Input number one"
+            }, {
+                "name": "input2",
+                "type": "complex",
+                "defaultValue": "null",
+                "label": "input #2",
+                "description": "Input number two"
+            }
+        ],
+        "outputs": [
+            {
+                "name": "output1",
+                "type": "complex",
+                "defaultValue": "null",
+                "label": "Output #1",
+                "description": "Output number one"
+            }
+        ]
+    },
+    "authors": [
+        {
+            "name": "Ignacio Pascual",
+            "email": "pascual@itis.swiss"
+        }, {
+            "name": "Katie Zhuang",
+            "email": "zhuang@itis.swiss",
+            "affiliation": "University of Great America"
```

Comment:
Bitte

Reply:
what? xD

Reply:
this is only for testing, to prefill the form